src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,234 @@
+    sample_color_xenograft: 'pink',
+};
+
+/**
+ * Object mapping of keys to objects indicating
+ * whether an attribute is primary or secondary,
+ * the formatter to display the inputted value t,
+ * and font coloring based on the key and value.
+ */
+export const config = {
+    orderMap: {
+        SEX: 1,
+        GENDER: 1,
+        AGE: 2,
+        CANCER_TYPE: 3,
+        CANCER_TYPE_DETAILED: 4,
+        KNOWN_MOLECULAR_CLASSIFIER: 5,
+        HISTOLOGY: 6,
+        OS_STATUS: 7,
+        OS_MONTHS: 8,
+        DFS_STATUS: 9,
+        DFS_MONTHS: 10,
+    },
+    keys: {
+        AGE: {
+            leadingComma: true,
+            primary: true,
+            formatter: t => {
+                return `${t} years old`;
+            },
+        },
+        CANCER_TYPE: {
+            primary: true,
+            hasSecondary: true,
+        },
+        CANCER_TYPE_DETAILED: {
+            primary: false,
+            formatter: t => {
+                return `${t}`;
+            },
+        },
+        DERIVED_NORMALIZED_CASE_TYPE: {
+            leadingComma: true,
+            primary: true,
+            values: {
+                Primary: colors.sample_color_primary,
+                Progressed: colors.sample_color_recurrence,
+                Recurrence: colors.sample_color_recurrence,
+                Metastasis: colors.sample_color_metastasis,
+            },
+        },
+        DERIVED_SAMPLE_LOCATION: {
+            primary: false,
+            formatter: t => {
+                return `${t}`;
+            },
+        },
+        DFS_STATUS: {
+            primary: true,
+            hasSecondary: true,
+            values: {
+                _: '#f00',
+                DiseaseFree: 'rgb(0, 128, 0)',
+                Yes: 'rgb(0, 128, 0)',
+            },
+        },
+        DFS_MONTHS: {
+            primary: false,
+            formatter: t => {
+                return `${t} months`;
+            },
+            trailingComma: false,
+        },
+        ERG_FUSION_ACGH: {
+            primary: true,
+            formatter: t => {
+                return `ERG-fusion aCGH: ${t}`;
+            },
+        },
+        GLEASON: {
+            primary: true,
+            formatter: t => {
+                return `Gleason: ${t}`;
+            },
+        },
+        PATIENT_DISPLAY_NAME: {
+            primary: false,
+            formatter: t => {
+                return `${t}`;
+            },
+        },
+        OS_MONTHS: {
+            primary: false,
+            formatter: t => {
+                return `${t} months`;
+            },
+        },
+        OS_STATUS: {
+            primary: true,
+            hasSecondary: true,
+            values: {
+                DECEASED: '#f00',
+                DEAD: '#f00',
+                LIVING: 'rgb(0, 128, 0)',
+                ALIVE: 'rgb(0, 128, 0)',
+            },
+        },
+        SAMPLE_DISPLAY_NAME: {
+            primary: false,
+            formatter: t => {
```

Comment:
might as well just leave formatter out when it just prints itself.  i believe you already handle case of no formatter in logic